README.md

```diff
@@ -4,18 +4,17 @@
 
 # Interface Exporter Action
 
-Interface Exporter Action automates the process of extracting TypeScript interfaces from Solidity contracts and provides compatibility with TypeChain. Developers can seamlessly generate typings with only a few lines of yaml code.
+Interface Exporter Action automates the process of extracting TypeScript interfaces from Solidity contracts and interfaces and provides compatibility with TypeChain. Developers can seamlessly generate typings with only a few lines of yaml code.
```

Comment:
If we decide to call the action _**Solidity Exporter Action**_ this paragraph needs to be changed